CoreData/VICoreDataManager/VIManagedObjectMapperSKZ.m

```diff
@@ -25,11 +27,39 @@ @implementation VIManagedObjectMapperSKZ
 + (instancetype)mapperWithUniqueKey:(NSString *)comparisonKey andMaps:(NSArray *)mapsArray;
 {
     VIManagedObjectMapperSKZ *mapper = [[self alloc] init];
+    NSUInteger mapCount = mapsArray.count;
+    
```

Comment:
I'd also probably break this whole block into it's own method that can be tested separately. I'd like to show you how our Specta integration works, we'll still need to keep writing those, albeit rarely. We'll need to learn Jest or some other JS testing framework and switch over to that soon. That doesn't really apply here though, since the CDM is tested with XCTest, not with Specta.